File: lv0/sum_of_even_number/sum_of_even_number.py
def solution(n):
    answer = 0
    for i in range(1, n+1) :
        if i % 2 == 0:
            answer += i
            
    return answer

assert solution(10) == 30, f"solution(10) == 30은 틀립니다. solution(10)의 반환 값은 {solution(10)}"

File: lv0/sum_of_even_number/sum_of_even_number.js
function solution(n) {
    let answer = 0;
    for (let i = 1; i <= n; i++) {
        if (i % 2 === 0) {
            answer += i;
        }
    }
    return answer;
}

let result = solution(10);
console.assert(result === 30, `solution(10) === 30 is incorrect. The return value of solution(10) is ${result}.`);
